drake/core/Path.h.in

```diff
@@ -0,0 +1,10 @@
+/* This file is auto-generated by CMAKE.  Do not edit. */
+
+#include <string>
+
+namespace Drake {
+  const std::string& getDrakePath() {
+    static std::string root("@PROJECT_SOURCE_DIR@");
+    return root;
```

Comment:
I would have written this as

    return "/path";

Then let the compiler optimize it.  Is it documented that declaring a local static variable is more efficient?